### DISCUSSION TITLE
Hide "Database Only" tables in "Content" navigation bar

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [directus/directus#12740] [Feature Requests]
Assuming you're logged in as an admin user, the following issues exists:

- When a collection is "Database Only", it shows up in the **Content** module navigation bar, even though in **Settings > Data Model** its dropdown has a "Make Collection Visible" entry that would imply it should be hidden. 🤔
- When you click the "Make Collection Visible" dropdown entry for a "Database Only" collection in **Settings > Data Model**, it makes the collection no longer "Database Only" and keeps the "Hidden" setting for the collection enabled. So the confusing result is that now after clicking "Make Collection Visible" the collection has gone from visible to hidden. 🤯

## Proposed Solution

1. If a table is "Database Only", don't show it in the **Content** module at all. **OR**
   Add a setting to toggle whether "Database Only" collections are shown in the content module.
2. For "Database Only" collections, remove the "Make Collection Visible" entry from the **Settings > Data Model** dropdown. **OR**
   Rename "Make Collection Visible" to "Add to Data Model" and change the icon, better indicating what it does.

**Comment by AaronBeaudoin** [comment]
The first part of this solution is especially valuable to me. Initially I was pushing for #11828 in order to hide my hundreds of "Database Only" tables from Directus completely, but now I'm leaning towards this as the more straightforward solution since really it seems more like an oversight that "Database Only" collections show up in the **Content** module in the first place.

**Comment by rijkvanzanten** [comment]
Thoughts @benhaynes?

**Reply by benhaynes** [reply]
I think that we should _not_ show database only tables in the Content Nav. You can then allow-list adding them within Data Model. This is good for temporary tables (etc) that you may not want to suddenly pop-in, and avoids the double toggle situation outlined in the OP.

**Reply by rijkvanzanten** [reply]
Agreed

**Comment by rijkvanzanten** [comment]
Implemented in https://github.com/directus/directus/pull/12743
